README.md

```diff
@@ -90,7 +90,7 @@ helm install ...\
 ```
 
 > You can also specify options 
-> for the on demand config: `onDemandRateLimitInterval`, `onDemandRateLimitBurst` and `onDemandAsk`
+> for the on-demand config: `onDemandRateLimitInterval`, `onDemandRateLimitBurst`, and `onDemandAsk`
```

Comment:
English is not main language (happy to learn that i'm wrong), but seems that comma is not required here https://www.grammarly.com/blog/comma-before-and/

Reply:
Yes. made that change